src/main/flight/pid.h

```diff
@@ -170,6 +170,8 @@ typedef struct pidProfile_s {
     uint8_t motor_output_limit;             // Upper limit of the motor output (percent)
     int8_t auto_profile_cell_count;         // Cell count for this profile to be used with if auto PID profile switching is used
     uint8_t transient_throttle_limit;       // Maximum DC component of throttle change to mix into throttle to prevent airmode mirroring noise
+    uint8_t ff_boost;                       // amount of high-pass filtered FF to add to FF, 100 means 100% added
+    uint16_t ff_boost_hz;                   // high-pass filter value for FF boost
```

Comment:
Adding elements to a parameter group array requires an increment in the version number, see https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md

Also, these should go to the very end of the struct.